src/OrbitGl/include/OrbitGl/CaptureViewElement.h

```diff
@@ -164,13 +164,24 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
   [[nodiscard]] virtual EventResult OnMouseEnter();
   [[nodiscard]] virtual EventResult OnMouseLeave();
 
+  [[nodiscard]] virtual bool RestrictDrawingToBody() const { return false; }
```

Comment:
I'm wondering if the default shouldn't be `true`. I guess that would make the change quite a bit bigger, because so many capture view elements need to override this right?

Reply:
I think the default should be false since it can have severe performance- and memory-implications if it is overused. I can add a comment to make this clearer. Now that you mention it, maybe I should consider a different name that sounds less like it should be the standard behavior.  Maybe `RequestSeparateLayer` instead?